modules/tinymce/src/core/test/ts/browser/paste/PasteTest.ts

```diff
@@ -148,7 +148,7 @@ describe('browser.tinymce.core.paste.PasteTest', () => {
     TinySelections.setSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
 
     editor.execCommand('mceInsertClipboardContent', false, { text: ' a ' });
-    TinyAssertions.assertContent(editor, '<p>t&nbsp;a&nbsp;xt</p>');
+    TinyAssertions.assertContent(editor, '<p>t a xt</p>');
```

Comment:
What happened here why is this now a space not a nbsp?

Reply:
@spocke It seems to be editor behavior to convert any `&nbsp;` in the content to regular spaces after an `input` event is dispatched.
<img width="499" alt="image" src="https://github.com/tinymce/tinymce/assets/56149628/e3ef7cde-0e32-44e0-9989-92a40d3d5fc0">

Reply:
The line that normalizes `&nbsp;` on `input` is here: https://github.com/tinymce/tinymce/blob/b4c1a77689f2cd7cdd65bd4456f7de405430f658/modules/tinymce/src/core/main/ts/keyboard/InputKeys.ts#L8

Reply:
Right that makes sense now.